MAVProxy/modules/mavproxy_map/mp_elevation.py

```diff
@@ -23,15 +23,15 @@
 class ElevationModel():
     '''Elevation Model. Only SRTM for now'''
 
-    def __init__(self, database='SRTM3', offline=0, debug=False):
+    def __init__(self, database='SRTM3', cachedir=None, offline=0, debug=False):
```

Comment:
please add the new parameter at the end so existing users that do EvelationModel("SRTM", True) still work